dsl_parser/exceptions.py

```diff
@@ -106,6 +106,8 @@ class UnknownSysPropertyError(Exception):
 
 
 class DSLParsingException(Exception):
+    hide_traceback = True
```

Comment:
From someone who likes grepping through the logs a lot: this sounds rather arbitrary, maybe we could make this parameter configurable (one day)?

Reply:
OK, now we do include the full traceback in the log. Just not in `execution.error` (as returned by rest-client)